src/components/dialog/association-dialog.js

```diff
@@ -248,7 +248,8 @@ class TccAssociationDialog extends BaseMixin(LitElement) {
 			this.dispatchEvent(new Event('association-dialog-save'));
 			this._close();
 		} else {
-			await this.requestUpdate();
+			this.requestUpdate();
+			await this.updateComplete();
```

Comment:
```suggestion
			await this.updateComplete;
```
this isn't a method, so you don't need to call it! See [here](https://github.com/lit/lit/blob/ff9d64eeff66f7b6c2a33283eb79d571f2e63728/packages/reactive-element/src/reactive-element.ts#L1322)